cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/util/CdcsdkContainer.java

```diff
@@ -50,6 +50,8 @@ public class CdcsdkContainer {
     // This line will be printed for each tablet so basically the count is equal to the total number
     // of tablets the CDCSDK Server is going to fetch the changes from.
     private int bootstrapLogLineCount = 1;
+    private String logMessageRegex = ".*Bootstrapping the tablet.*\\n";
```

Comment:
This log message only appears when we go for bootstrapping the tablet, which is fine for now because we do not have any tests for scenarios where bootstrapping is skipped.

I am thinking that we should replace this message with `Checkpoint for tablet` as this appears every time - doesn't matter if we bootstrap or not.

Reply:
I should check if that works. I made this field configurable because I want it to check for `Mapping table` on a restart.

Reply:
I have checked this `Checkpoint for tablet` works in my another PR #39 - let's change it to that only.